.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
```

Comment:
It seems `v4` is already available, see: https://github.com/actions/stale#usage

Reply:
moved to use v4.